Assets/Scripts/Bottle.cs

```diff
@@ -50,6 +51,56 @@ public void CreateBottle(int _height, GameObject[] _startLiquids, BottleManager
         bottleManager = _bottleManager;
     }
 
+    public void SetBottomOfBottle( GameObject bottomOfBottle )
```

Comment:
Use C# { get; set; } syntax

Reply:
I thought that you said using the C# short-hand for fields / properties would break the Unity SerializeField feature?